### ISSUE TITLE
module 'file_picker' not found

### ISSUE THREAD

**Issue by LeinHeart** [miguelpruivo/flutter_file_picker#265]
**Describe the bug**
Hi, I'm currently using file_picker version 1.9.0
but the error I occurs is from before the 1.9.0 version

so everytime I want to build to my iOS real device I caught this error

```
Error output from Xcode build:
↳
    2020-05-04 09:46:49.609 xcodebuild[11469:216052]  DTDeviceKit: deviceType from 00008030-001471222200802E was NULL
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/kristopherchayadi/Desktop/Flutter/sakura-mobile-phase2/sakura-mobile/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module
    'file_picker' not found
    @import file_picker;
     ~~~~~~~^~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the precompiled application for the device.

Error launching application on Kristopher’s iPhone.
```

But when I run it on XCode, it run smoothly.

the error occurs when I start to use my real iPhone, simulator works fine.

I already try flutter clean, restart my macbook, restart my iPhone, but still no luck

```
**Flutter Version details**
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.3 19D76, locale en-ID)
    • Flutter version 1.12.13+hotfix.9 at /Users/kristopherchayadi/developer/flutter
    • Framework revision f139b11009 (5 weeks ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/kristopherchayadi/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.0

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 37.0.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.44.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.9.1

[✓] Connected device (1 available)
    • Kristopher’s iPhone • 00008030-001471222200802E • ios • iOS 13.4

• No issues found!
```

Edit : when I try to archive my project, got popup Module 'file_picker' not found 

![Screen Shot 2020-05-04 at 09 58 48](https://user-images.githubusercontent.com/39891325/80933424-e5c6d480-8ded-11ea-8435-54e2ea030039.png)

**Comment by miguelpruivo** [comment]
You should need to remove your derived data from Xcode. 

You can go to File > Workspace Settings if you are in a workspace environment or File > Project Settings for a regular project environment.

Then click over the little grey arrow under Derived data section and select your project folder to delete it.

**Comment by miguelpruivo** [comment]
Have you managed to fix it @LeinHeart? Thanks!

**Comment by LeinHeart** [comment]
Hello, thank you for replying @miguelpruivo 
yes I fixed it by migrate the project for XCode 11.4
I think this is from the xcode side, not the file_picker side, since when I try to remove file_picker, the error still pops up but different packages
